addons/purchase_stock/models/stock.py

```diff
@@ -257,6 +257,17 @@ def _set_default_route_id(self):
             orderpoint_wh_supplier.route_id = route_id[0].id
         return super()._set_default_route_id()
 
+    def _get_min_qty_to_order(self):
```

Comment:
The name of the method is ambiguous, it should be `_get_qty_multiple_to_order` IMO.